shared/utils/add_stig_references.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python2
+
+try:
+    from xml.etree import cElementTree as etree
+except ImportError:
+    import cElementTree as etree
+
+import re
+import sys
+import argparse
+
+parser = argparse.ArgumentParser(
+    description='Add Stig references to XCCDF files.')
+parser.add_argument("reference", help="Path to DISA STIG Reference XCCDF")
```

Comment:
should be `"--disa-stig", help="DISA STIG Reference XCCDF file"`

Reply:
Let's do actual argument parsing as it will help to reduce confusion of the python script.